### DISCUSSION TITLE
能不能把绑定与uid合并呢，查uid自动绑定该账号，2.X版本的那种

### DISCUSSION THREAD

**Discussion by ID114514** [Arondight/Adachi-BOT#453] [General]
可以省的绑定改绑了，还能查询更多信息例如他人的圣遗物搭配之类的

**Comment by Arondight** [comment]
我不喜欢这样，目前会维持现状，别人的角色圣遗物现在也支持查询

**Reply by Arondight** [reply]
`100000001的胡桃`，`@本群群主的堂主`

**Reply by ID114514** [reply]
在不知道对方的米游社账号情况下应该如何查询圣遗物呢

**Reply by ID114514** [reply]
> `100000001的胡桃`，`@本群群主的堂主`

好的了解了
